src/components/ClipboardButton.tsx

```diff
@@ -12,7 +12,6 @@ const styles = {
   clickableLink: {
     display: 'inline',
     color: colors.accent(),
-    cursor: 'pointer',
```

Comment:
This cursor style is applied by `Clickable` and doesn't need to be respecified here.